centralserver/settings.py

```diff
@@ -182,18 +182,14 @@
     'django.contrib.humanize',
     'centralserver.contact',
     'kalite.updates',
-    'kalite.caching',
     'centralserver.i18n',
     'tastypie',
     'announcements',
-    'kalite.playlist',
```

Comment:
Mind you, there are only 330 of them, as they didn't end up using them very long.